config/webpack/webpack.config.js

```diff
@@ -121,9 +121,20 @@ const buildWebpackConfigs = builds.map(
 
     const internalJs = [paths.src, ...paths.compilePackages];
 
+    const entry = [paths.clientEntry];
+    const devEntries = [
+      `${require.resolve(
+        'webpack-dev-server/client'
+      )}?http://localhost:${port}/`
```

Comment:
Will this localhost reference work if localhost isn't in the list of allowed hosts?